Makefile

```diff
@@ -1,6 +1,6 @@
 GO ?= go
 BUILD ?= build
-REPO ?= fjogeleit/policy-reporter
+REPO ?= kyverno/policy-reporter
```

Comment:
This should be reverted by now. The new home for the images is another issue and I will work on it the upcoming week